asm/bss_8001C4D0.s

```diff
@@ -2135,7 +2135,7 @@ glabel gCupSelection # 0x8018EE09
 glabel D_8018EE0A
 .skip 1
 
-glabel gCourseSelection # 0x8018EE0B
+glabel gCupTrackSelection # 0x8018EE0B
```

Comment:
We need to be consistent and choose one of Course or Track. I would recommend sticking with Course since the rest of the repo is named that way. `gCupCourseSelection`

If track is better than course I can be convinced. But I've been thinking `course` for so long I'm biased towards that.

Reply:
No, you're right. Since they're synonyms I swap between them all the time and forgot that ```course``` is the more canonical term in this repo.